src/main/webapp/WEB-INF/tags/student/feedbackSubmissionEdit/submitButton.tag

```diff
@@ -0,0 +1,22 @@
+<%@ tag description="feedbackSubmissionForm.tag - Submit button" %>
+<%@ tag import="teammates.common.util.Const"%>
+<%@ taglib uri="http://java.sun.com/jsp/jstl/core" prefix="c" %>
+<%@ attribute name="preview" type="java.lang.Boolean" required="true" %>
+<%@ attribute name="submittable" type="java.lang.Boolean" required="true" %>
+
+<c:set var="TOOLTIP_EDIT_SAVE" value="<%=Const.Tooltips.FEEDBACK_SESSION_EDIT_SAVE%>"/>
+
+<c:choose>
+    <c:when test="${preview or (not submittable)}">
+        <input disabled="disabled" type="submit" class="btn btn-primary"
+               id="response_submit_button" data-toggle="tooltip"
+               data-placement="top" title="${TOOLTIP_EDIT_SAVE}"
+               value="Submit Feedback" style="background: #66727A;">
+    </c:when>
+    <c:otherwise>
+        <input type="submit" class="btn btn-primary"
+               id="response_submit_button" data-toggle="tooltip"
+               data-placement="top" title="${TOOLTIP_EDIT_SAVE}"
+               value="Submit Feedback">
```

Comment:
Likewise, you can just set an attribute for " style="background: #66727A;" based on the condition.